docs/source/api_reference/embed_widgets/circle_marker.html

```diff
@@ -0,0 +1,489 @@
+</script>
+<script type="application/vnd.jupyter.widget-state+json">
+{
+    "version_major": 2,
+    "version_minor": 0,
+    "state": {
+        "65a260cb0ac24108a8036e011f5193b8": {
+            "model_name": "LeafletTileLayerModel",
+            "model_module": "jupyter-leaflet",
+            "model_module_version": "~0.9.1",
+            "state": {
+                "_model_module_version": "~0.9.1",
+                "_view_count": null,
+                "_view_module_version": "~0.9.1",
+                "attribution": "Map tiles by <a href=\"http://stamen.com/\">Stamen Design</a>, under <a href=\"http://creativecommons.org/licenses/by/3.0\">CC BY 3.0</a>. Data by <a href=\"http://openstreetmap.org/\">OpenStreetMap</a>, under <a href=\"http://creativecommons.org/licenses/by-sa/3.0\">CC BY SA</a>.",
+                "min_zoom": 1,
+                "name": "Stamen.Watercolor",
+                "options": [
+                    "attribution",
+                    "detect_retina",
+                    "max_zoom",
+                    "min_zoom",
+                    "tile_size"
+                ],
+                "url": "http://stamen-tiles-a.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.png"
+            }
+        },
+        "1ccd183462354a0b91f2f4245ded977b": {
+            "model_name": "LayoutModel",
+            "model_module": "@jupyter-widgets/base",
+            "model_module_version": "1.1.0",
+            "state": {}
+        },
+        "0c255695e6294b2e92bc1a8af7688179": {
+            "model_name": "LeafletMapModel",
```

Comment:
You have four map models created in this HTML file. I think there was a problem when you generated them (Maybe you restarted the kernel multiple times and the widgets models were not cleaned properly).